### PR TITLE
Add-AppsandFeaturesEntries Elgato.CameraHub version 1.6.1.999

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/125371)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie

**Comment by stephengillie** [comment]
image

**Comment by stephengillie** [comment]
Hi @BrandonWanHuanSheng,

Sorry for the delay. We had a few different events collide into an unexpected backlog.

File: manifests/e/Elgato/CameraHub/1.6.1.999/Elgato.CameraHub.installer.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ InstallModes:
 - silent
 - silentWithProgress
 UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 1.6.1.999
```
Comment (stephengillie) [inline manifests/e/Elgato/CameraHub/1.6.1.999/Elgato.CameraHub.installer.yaml:17]:
```suggestion
- DisplayVersion: 1.6.1.999
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**